internal/validate/validate.go

```diff
@@ -97,15 +97,18 @@ var ErrMissingCredentials = errors.New("missing credentials")
 
 // Credentials validates public and private API keys have been set.
 func Credentials() error {
-	if config.PrivateAPIKey() == "" || config.PublicAPIKey() == "" {
-		return fmt.Errorf(
-			"%w\n\nTo set credentials, run: %s %s",
-			ErrMissingCredentials,
-			config.ToolName,
-			"config",
-		)
+	if config.AuthToken() != "" {
```

Comment:
[q] what is the reason for us to check `AuthToken` separately from `PrivateAPIKey` and `PublicAPIKey`?

Reply:
got it, thanks!